test/views/vat/WhichVATServicesToAddViewSpec.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.vat
+
+import play.api.data.Form
+import forms.vat.WhichVATServicesToAddFormProvider
+import models.vat.WhichVATServicesToAdd
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.vat.whichVATServicesToAdd
+
+class WhichVATServicesToAddViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "whichVATServicesToAdd"
+
+  val form = new WhichVATServicesToAddFormProvider()()
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView = () => whichVATServicesToAdd(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => whichVATServicesToAdd(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  "WhichVATServicesToAdd view" must {
+    behave like normalPage(createView, messageKeyPrefix)
```

Comment:
add content tests

Reply:
The only content on the page is the header and radio buttons, both have tests already.